flow/project.py

```diff
@@ -2519,13 +2768,13 @@ def run(self, jobs=None, names=None, pretend=False, np=None, timeout=None, num=N
         :param order:
             Specify the order of operations, possible values are:
                 * 'none' or None (no specific order)
-                * 'by-job' (operations are grouped by job)
```

Comment:
We can't just remove this, this is part of the public API. Why can we no longer support this?

Reply:
Aggregates are created on a per-group basis. We'd now have to iterate over aggregates hence we the `by-job` order won't be the default choice.

Reply:
This was discussed on slack, can you summarize the results of that discussion here, please?

Reply:
I will do that

Reply:
The result of the discussion of slack on this topic is posted here
https://github.com/glotzerlab/signac-flow/pull/335#issuecomment-677747896